geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/netty/Coder.java

```diff
@@ -421,6 +422,14 @@ public static boolean isInfinity(byte[] bytes) {
     return isPositiveInfinity(bytes) || isNegativeInfinity(bytes);
   }
 
+  // Checks if the given byte array is equivalent to the String "NaN", ignoring case.
+  public static boolean isNaN(byte[] bytes) {
+    if (bytes == null) {
+      return false;
+    }
```

Comment:
This null check is not necessary, as `equalsIgnoreCaseBytes()` handles nulls.

Reply:
Would it therefore also not be necessary in isPositiveInfinity() and isNegativeInfinity() below it?

Reply:
Yeah, good catch, thanks for cleaning that up.